website/i18n/zh-cn/docusaurus-theme-classic/footer.json

```diff
@@ -25,7 +25,7 @@
   },
   "link.item.label.专题详解": {
     "message": "专题详解",
-    "description": "The label of footer link with label=专题详解 linking to /docs/guides/features/runtime/code-spliting"
+    "description": "The label of footer link with label=专题详解 linking to /docs/guides/features/runtime/code-splitting"
```

Comment:
The `code-spliting.md` should be renamed too.

Reply:
Yes, I renamed it.